modules/ROOT/pages/6.4.2-release-notes.adoc

```diff
@@ -76,6 +76,16 @@ include::partial$misc/admon-releasenotes-for-stable.adoc[]
 
 // {productname} 6.4.2 also includes the following bug fixes:
 
+=== Tab navigation no longer incorrectly stops at menu buttons within toolbar groups.
+
+In previous versions of {productname}, having a menu button in the “more” floating toolbar, would cause issues with the navigation behaviour.
+
+As a consquence, Tabbing would stop at the menu button, rather than stopping at the toolbar groups **first** button.
+
+To fix this issue, {productname} removed `tabstopping` capability for menu buttons when they are rendered inside toolbar.
```

Comment:
```suggestion
To fix this issue, {productname} removed *tabstopping* capability for menu buttons when they are rendered inside toolbar.
```
Copy-edit.

Also: two questions.

1. _rendered inside toolbar_ is missing a word.
	Is it the indefinite article — _rendered inside **a** toolbar_.
	Or a qualifying adjective — _rendered inside **the floating** toolbar_ or _rendered inside **a floating** toolbar_
	Or something else?

2. Is *tabstopping* a term of local art or a neologism?
	If it’s a neologism, is there a longer string that describes the removed capability, especially a longer string used by developers to describe the capability?

Reply:
@brian-forte will answer the above questions in the current conversation thread.